### DISCUSSION TITLE
Support dragging & dropping columns in Data Explorer

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2275] [Feature Ideas]
We'd like to support using drag and drop to make building explorations in the Data Explorer easier and more intuitive.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by jeromeperez** [comment]
Hi
Oh yes very interesting feature to make building exploration **and to reorganize order of field in existing exploration**

**Reply by seancolsen** [reply]
Thanks — good to know you're interested in this, @jeromeperez.
